src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java

```diff
@@ -1,21 +1,64 @@
+import static org.folio.marccat.config.constants.Global.BOOK_TYPE;
+import static org.folio.marccat.config.constants.Global.COLOR_CODE;
+import static org.folio.marccat.config.constants.Global.COMPUTER_TYPE;
+import static org.folio.marccat.config.constants.Global.CONFIGURATION_CODE;
+import static org.folio.marccat.config.constants.Global.CONTROL_FIELD_CATEGORY_CODE;
+import static org.folio.marccat.config.constants.Global.DIMENSION_CODE;
+import static org.folio.marccat.config.constants.Global.ELECTRONIC_RESOURCE;
+import static org.folio.marccat.config.constants.Global.EMPTY_VALUE;
+import static org.folio.marccat.config.constants.Global.FIXED_FIELDS;
+import static org.folio.marccat.config.constants.Global.FORM_OF_ITEM_CODE;
+import static org.folio.marccat.config.constants.Global.FORM_OF_MATERIAL_LABEL;
+import static org.folio.marccat.config.constants.Global.GLOBE;
+import static org.folio.marccat.config.constants.Global.GOVERNMENT_PUBLICATION_CODE;
+import static org.folio.marccat.config.constants.Global.KIT_CODE;
+import static org.folio.marccat.config.constants.Global.MAP_CODE;
+import static org.folio.marccat.config.constants.Global.MAP_TYPE;
+import static org.folio.marccat.config.constants.Global.MATERIAL_TAG_CODE;
+import static org.folio.marccat.config.constants.Global.MEDIUM_FOR_SOUND_CODE;
+import static org.folio.marccat.config.constants.Global.MICROFORM;
+import static org.folio.marccat.config.constants.Global.MIXED_TYPE;
+import static org.folio.marccat.config.constants.Global.MOTION_PICTURE;
+import static org.folio.marccat.config.constants.Global.MUSIC_TYPE;
+import static org.folio.marccat.config.constants.Global.NON_PROJECTED_GRAPHIC;
+import static org.folio.marccat.config.constants.Global.NOTATED_MUSIC;
+import static org.folio.marccat.config.constants.Global.OTHER_MATERIAL_TAG_CODE;
+import static org.folio.marccat.config.constants.Global.PHYSICAL_DESCRIPTION_TAG_CODE;
+import static org.folio.marccat.config.constants.Global.PHYSICAL_TYPES_MAP;
+import static org.folio.marccat.config.constants.Global.POLARITY_CODE;
+import static org.folio.marccat.config.constants.Global.PROJECTED_GRAPHIC;
+import static org.folio.marccat.config.constants.Global.REMOTE_SENSING_IMAGE;
+import static org.folio.marccat.config.constants.Global.SERIAL_TYPE;
+import static org.folio.marccat.config.constants.Global.SOUND_ON_MEDIUM_OR_SEPARATE_CODE;
+import static org.folio.marccat.config.constants.Global.SOUND_RECORDING;
+import static org.folio.marccat.config.constants.Global.SPECIFIC_MATERIAL_DESIGNAION_ON_CODE;
+import static org.folio.marccat.config.constants.Global.TACTILE_MATERIAL;
+import static org.folio.marccat.config.constants.Global.TEXT_CODE;
+import static org.folio.marccat.config.constants.Global.UNSPECIFIED;
+import static org.folio.marccat.config.constants.Global.VIDEO_RECORDING;
+import static org.folio.marccat.config.constants.Global.VISUAL_TYPE;
+import static org.folio.marccat.enumaration.CodeListsType.*;
```

Comment:
It's generally frowned upon to use wildcard includes.

Reply:
I have changed it.